plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/DigestExtractor.java

```diff
@@ -33,7 +34,8 @@
  *
  * @since 3.0
  */
-class DigestExtractor
+@ThreadSafe
```

Comment:
No reason except documentation. But yes, even this PR is inconsistent with documenting thread-safe classes, so removing this to lessen confusion would be ok.